MotionMark/resources/debug-runner/debug-runner.js

```diff
@@ -528,18 +528,20 @@ window.suitesManager = {
     }
 }
 
-Utilities.extendObject(window.benchmarkController, {
-    initialize: async function()
+class DebugBenchmarkController extends BenchmarkController {
+    async initialize()
     {
         document.title = Strings.text.title.replace("%s", Strings.version);
         document.querySelectorAll(".version").forEach(function(e) {
             e.textContent = Strings.version;
         });
+        
+        this.graphController = new GraphController;
 
-        document.forms["benchmark-options"].addEventListener("change", benchmarkController.onBenchmarkOptionsChanged, true);
-        document.forms["graph-type"].addEventListener("change", benchmarkController.onGraphTypeChanged, true);
-        document.forms["time-graph-options"].addEventListener("change", benchmarkController.onTimeGraphOptionsChanged, true);
-        document.forms["complexity-graph-options"].addEventListener("change", benchmarkController.onComplexityGraphOptionsChanged, true);
+        document.forms["benchmark-options"].addEventListener("change", () => { this.onBenchmarkOptionsChanged() }, true);
```

Comment:
Another way to write this:
```
document.forms["benchmark-options"].addEventListener("change", this.onBenchmarkOptionsChanged.bind(this), true);
```

Reply:
I find arrow functions nicer than `.bind()`